contrib/functions/go/generate-blueprint-docs/README.md

```diff
@@ -0,0 +1,164 @@
+# generate-blueprint-docs
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `generate-blueprint-docs` function generates opinionated documentation for a [blueprint](https://github.com/GoogleCloudPlatform/blueprints).
```

Comment:
It may be helpful to call out that blueprint is also known as kpt package.

Reply:
Reworded this as `blueprint kpt package`. The readme format itself is pretty generic and does support native k8s resources. I feel this is general enough we could even make this a standardized opinionated readme format for all kpt packages as `generate-kpt-pkg-docs`.

Reply:
SGTM. This is a good starting point.
If we want to rename this fn to `generate-kpt-pkg-docs`, we should do it now.

Reply:
Renamed to `generate-kpt-pkg-docs`